MapboxDirections/MBRouteStep.swift

```diff
@@ -547,7 +547,7 @@ public class RouteStep: NSObject, NSSecureCoding {
         self.coordinates = coordinates
     }
     
-    internal convenience init(json: JSONDictionary) {
+    public convenience init(json: [ String: Any ]) {
```

Comment:
`JSONDictionary` is a private typealias in MBDirections:

> MBRouteStep.swift:570:24: Initializer cannot be declared public because its parameter uses an internal type

Using `init(json: JSONDictionary` in external tests is prohibited by that. Not sure how to make `typealias` public (they should be by default?), but the easier solution was to change the function signature without the typealias